source/docs/contribute/backend.md

```diff
@@ -7,4 +7,29 @@ This page should contain:
 
 - How to get started.
 - What branch to target.
-- How to test locally.
\ No newline at end of file
+- How to test locally.
+
+## Finding the backend repository
+
+The backend repository is hosted @ https://github.com/hacs/integration
+
+## Target branch for contributor PRs
+
+When contributing to the backend, target your PR against the `{{ config.remote_branch }}` branch.
+
+## Creating your code changes using the devcontainer
+
+1. Spin up the [devcontainer](/docs/contribute/devcontainer.md).
+
+2. Add your code changes.
+3. To start a preview of the application, issue the following command:
+
+    ```bash
+    scripts/develop
+    ```
+
+    - You have now started a local server that hosts the application on `http://localhost:8123`
+
+    - The server automatically reloads to reflect your changes.

```

Comment:
```suggestion
```

This is currently not possible, and I doubt it ever will be, but if it was a lot of custom integration devs would be happy.